x-pack/plugins/reporting/server/lib/event_logger/logger.ts

```diff
@@ -27,171 +30,166 @@ export interface ExecutionCompleteMetrics {
   byteSize: number;
 }
 
+export interface IReportingEventLogger {
+  logEvent(message: string, properties: LogMeta): void;
+  startTiming(): void;
+  stopTiming(): void;
+}
+
+const getEventLog = (logger: LevelLogger, lProperties: Partial<LogMeta>) => {
+  return new EcsLogAdapter(logger, lProperties);
+};
+
 /** @internal */
-export function reportingEventLoggerFactory(eventLog: IEventLogService) {
-  const genericLogger = eventLog.getLogger({ event: { provider: PLUGIN_ID } });
+export function reportingEventLoggerFactory(logger: LevelLogger) {
+  const genericLogger = getEventLog(logger, { event: { provider: PLUGIN_ID } });
```

Comment:
Why don't we instantiate that directly? The `getEventLog` function seems redundant in a way.

```suggestion
  const genericLogger = new EcsLogAdapter(logger, { event: { provider: PLUGIN_ID } });
```